lib/node_modules/@stdlib/math/base/special/cphase/lib/native.js

```diff
@@ -0,0 +1,47 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2023 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var addon = require( './../src/addon.node' );
+
+
+// MAIN //
+
+/**
+* Computes the argument of a complex number in radians.
+*
+* @private
+* @param {Complex128} z - complex number
+* @returns {number} argument
+*
+* @example
+* var Complex128 = require( '@stdlib/complex/float64' );
+* var phi = cphase( new Complex128( 5.0, 3.0 ) );
```

Comment:
```suggestion
*
* var phi = cphase( new Complex128( 5.0, 3.0 ) );
```

Reply:
cc @Planeshifter We should be able to create a lint rule for this, correct?

@steff456 Would you mind opening an internal todo for this? Thanks!